### ISSUE TITLE
Only noise in eval mode

### ISSUE THREAD

**Issue by orionpeng** [r9y9/wavenet_vocoder#90]
Hi all,
I am training a wavenet model from scratch. 
After 20K steps training,  I find a problem that the predicted audios from both test_eval and train_eval folders are just noises. However, the quality of the predicted audios generated from ground truth Mel-Spectrum is quite good. 
Any suggestions? 
Thx

**Comment by orionpeng** [comment]
Maybe the training time is not enough, I will wait for more time to see the results.

**Comment by r9y9** [comment]
From my experience, MoL WaveNet takes over 1000k training steps to produce good quality of speech.

**Comment by orionpeng** [comment]
@r9y9 Yes, thx for your comment.
After 60k training steps, I can hear some speech-like voice from the predicted audios.
